bundle/src/main/java/com/adobe/acs/commons/indesign/dynamicdeckdynamo/services/impl/DynamicDeckConfigurationServiceImpl.java

```diff
@@ -24,7 +24,7 @@
 import org.osgi.service.component.annotations.Component;
 import org.osgi.service.metatype.annotations.Designate;
 
-@Component(service = DynamicDeckConfigurationService.class)
+@Component(service = DynamicDeckConfigurationService.class, immediate = true)
```

Comment:
I guess we can leave this - but you definitely shouldn't need it to slingscripthelper to get the service... might be something weird going on w/ the MCP forms